src/cmdlib.sh

```diff
@@ -206,27 +210,36 @@ runcompose() {
     # Implement support for automatic local overrides:
     # https://github.com/coreos/coreos-assembler/issues/118
     local overridesdir=${workdir}/overrides
-    if [ -d "${overridesdir}"/rpm ]; then
-        (cd "${overridesdir}"/rpm && createrepo_c .)
-        echo "Using RPM overrides from: ${overridesdir}/rpm"
-        local tmp_overridesdir=${TMPDIR}/override
+    local tmp_overridesdir=${TMPDIR}/override
+    local override_manifest="${tmp_overridesdir}"/coreos-assembler-override-manifest.yaml
+    if [ -d "${overridesdir}"/rpm ] || [ -n "${ref_is_temp}" ]; then
         mkdir "${tmp_overridesdir}"
-        cat > "${tmp_overridesdir}"/coreos-assembler-override-manifest.yaml <<EOF
+        cat > "${override_manifest}" <<EOF
 include: ${workdir}/src/config/manifest.yaml
-repos:
-  - coreos-assembler-local-overrides
 EOF
         # Because right now rpm-ostree doesn't look for .repo files in
         # each included dir.
         # https://github.com/projectatomic/rpm-ostree/issues/1628
         cp "${workdir}"/src/config/*.repo "${tmp_overridesdir}"/
+        manifest=${override_manifest}
+    fi
+    if [ -n "${ref_is_temp}" ]; then
+        echo 'ref: ''"'"${ref}"'"' >> "${override_manifest}"
```

Comment:
The quoting here is a bit hard ... what's the need for `open`ref: `close``open`"`close`"${ref}"`open``close`?

Reply:
i think he want's to pass double quotes into the file.. i.e. we need to write `ref: "tmpref-fedora-coreos"`

Reply:
Yep